### PR TITLE
Add bee in courtesy to DaddyDiddy

### PR DESCRIPTION
**Pull request by Nilau1998**
Diddy made an extensive guide on bees for GTNH specifically, as a return for his effort I implemented a new bee. His bee will pretty much be a mix of Industrious and Imperial w/ a nice side effect of producing seed oil. You could almost say it makes alvearies right away.

His guide: https://youtube.com/playlist?list=PLIbgty4btOhU2RYv735pG8QfCPB056jJp&si=YxUxvc1uBM_mP1Ho

What does makes this bee special?
It basically provides the ability to produce all alveary components out of the box and has some interesting side effects.
- It needs an *active* GT machine below the Apiary to even have a chance to breed, to bee a bit extra toxic it has a 1% to breed. Diddy himself: "I'm fine with it, because that's the level of grinding for bees i do on my stream".
- A new passive effect, Machine Booster: The bee learned about the inner workings of GT machines. Now it can provide a 2s boost to whatever GT singleblock machine under EV that it can find within its range. Example, if it finds a Macerator grinding away on ores, it will apply the boost and the recipe skips 2 seconds immediatly. The recipe can only be skipped if its longer than 3 seconds so 0 ticking is prevented. The effect is additionally throttled to only tick every 30 seconds. Which reduces the bonus of it massively as it pretty much barely ever happens.
- Flowertype Fire, the bee needs fire to power its alveary prodution and to give out the Machine Booster passive.

The odds of the effect being applied are:
grafik

### CODE REVIEW

File: src/main/java/gregtech/common/items/ItemComb.java [hunk 1]
```diff
@@ -4,14 +4,7 @@
...
 import static gregtech.api.enums.GT_Values.V;
-import static gregtech.api.enums.Mods.AE2FluidCraft;
-import static gregtech.api.enums.Mods.ExtraUtilities;
-import static gregtech.api.enums.Mods.GalaxySpace;
-import static gregtech.api.enums.Mods.GregTech;
-import static gregtech.api.enums.Mods.MagicBees;
-import static gregtech.api.enums.Mods.NewHorizonsCoreMod;
-import static gregtech.api.enums.Mods.Thaumcraft;
-import static gregtech.api.enums.Mods.ThaumicBases;
+import static gregtech.api.enums.Mods.*;
```
Comment (Dream-Master) [inline src/main/java/gregtech/common/items/ItemComb.java:7]: please change this back. We not use * imports

File: src/main/java/gregtech/loaders/misc/bees/GT_Flowers.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
+package gregtech.loaders.misc.bees;
+
+import forestry.api.apiculture.FlowerManager;
+import forestry.api.genetics.*;
```
Comment (Dream-Master) [inline src/main/java/gregtech/loaders/misc/bees/GT_Flowers.java:4]: same here

**Review by Dream-Master — DISMISSED** [review]

File: src/main/java/gregtech/common/items/ItemComb.java [hunk 3]
```diff
@@ -1311,6 +1304,18 @@ public void initCombsRecipes() {
                 new int[] { 100 * 100 },
                 Voltage.LV);
         }
+        addCentrifugeToItemStack(
```
Comment (Dream-Master) [inline src/main/java/gregtech/common/items/ItemComb.java:1307]: this looks very common recipe. can you add a special item, item part maybe ask Diddi for an idea

**Comment by Dream-Master** [comment]
@Nilau1998

**Comment by Dream-Master** [comment]
This need more discussion

**Comment by Pxx500** [comment]
This doesn't look too OP, can we merge it or is there still something to discuss?

**Comment by Dream-Master** [comment]
@Nilau1998 i open it and afte the changes we can merge this

File: src/main/java/gregtech/loaders/misc/GT_BeeDefinition.java [hunk 4]
```diff
@@ -1661,6 +1662,25 @@ public void accept(GT_BeeDefinition dis) {
             tMutation.requireResource(GameRegistry.findBlock(ExtraCells2.ID, "walrus"), 0);
         }
     }),
+    DIDDY(GT_BranchDefinition.ORGANIC, "Diddy", true, new Color(85, 37, 130), new Color(253, 185, 39), beeSpecies -> {
+        beeSpecies.addProduct(GT_Bees.combs.getStackForType(CombType.DIDDY), 0.2f);
+        beeSpecies.setHumidity(EnumHumidity.NORMAL);
+        beeSpecies.setTemperature(NORMAL);
+        beeSpecies.setHasEffect();
+    }, template -> {
+        AlleleHelper.instance.set(template, SPEED, GT_Bees.speedBlinding);
```
Comment (Volence) [inline src/main/java/gregtech/loaders/misc/GT_BeeDefinition.java:1671]: Only thing I'm slightly concerned about is how early this makes getting the "Blinding" stat through this bee. Iirc the first time you can get it before this is the stardust bee which is a bit off from this and gives a bit more of a challenge to get that.

Comment (FourIsTheNumber) [reply src/main/java/gregtech/loaders/misc/GT_BeeDefinition.java:1671]: This bee already has enough going for it without having all of the best stats.

Comment (Dream-Master) [reply src/main/java/gregtech/loaders/misc/GT_BeeDefinition.java]: changed to super speed

**Review by boubou19 — APPROVED** [review]

**Pull request merged**